content/_includes/post.macros.njk

```diff
@@ -609,3 +632,82 @@ params:
     </ul>
   {% endif %}
 {% endmacro %}
+
+{# @docs
+label: page_nav
+category: pagination
+note: Navigate paginated collections
+params:
+  title:
+    type: string
+    note: Will be hidden, for screen-readers only
+  pagination:
+    type: 11ty pagination object
+    default: none
+  current:
+    type: boolean
+    default: false
+    note: Show the current location in pagination?
+  id:
+    type: string
+    default: ''
+    note: |
+      When repeating the pagination on a page,
+      each instance should get a unique ID
+  level:
+    type: integer [1-6]
+    default: 2
+    note: Heading level
+#}
+{% macro page_nav(title, pagination, current=true, id='', level=2) %}
+{% if pagination %}
+{% set id = ['resource-nav-title', id] | join('-') %}
+<nav class="pagination" aria-labeledby="{{ id }}">
+  {% h level, {
+    'id': id,
+    'class': 'sr-only'
+  } %}
+    {{ title | mdInline | safe }}
+  {% endh %}
+  <p>
```

Comment:
I wonder if we might use the nav element here instead of a `<p>`? That's what I mostly see on some accessibility sites but there are always exceptions based on the other content on the page.

Also, I don't know if I am reading this correctly, https://www.w3.org/TR/wai-aria-practices-1.2/#general-principles-of-landmark-design but it seems like we could use the heading as the label since it is in the same container and it preceeds the nav... so, for instance 2019 Resources could be used instead of including a duplicate header for sr-only. Let me know if I am reading this wrong. 
![image](https://user-images.githubusercontent.com/1581694/86705608-ada16500-bfdb-11ea-9b53-b155a5c04f7b.png)

Reply:
- We are using the `nav`, just with a `p` inside it. We could remove the `p` (I think it's actually just in the way), but we don't need to replace it.
- oh, yeah, you're right about the header. :) I'll fix that.

Reply:
@stacyk I think I've resolved all these issues, if you want to confirm